content/blog/pulumi-for-aws-automate-secure-manage/index.md

```diff
@@ -0,0 +1,87 @@
+
+AWS Cloud Development Kit (CDK) on Pulumi is now generally available. This powerful integration includes:
+
+* Full support for CDK features like assets, custom resources, aspects, and context values/methods.
+* Coverage for 19 of the top 20 construct libraries, thoroughly validated through testing.
+* Support for Docker assets, so you can build/push Docker images to ECR, streamlining  the delivery of containerized applications.
+
+[Read more about AWS CDK on Pulumi](/blog/aws-cdk-on-pulumi-1.0/)
+
+## Beyond Pulumi IaC
+
+Over the last few years, our customers have shared a clear message:  IaC alone is insufficient. The complexities of modern cloud environments demand more than just IaC to manage cloud resources, improve security, and ensure compliance. In response, Pulumi's offerings have expanded into a comprehensive suite of tools that go beyond IaC.
+
+![Pulumi platform diagram](platform.png)
+
+### Pulumi Insights: Intelligent Cloud Management
+
+[Pulumi Insights](/product/pulumi-insights/) empowers organizations to tackle cloud complexity by delivering visibility, actionable insights, and AI-assisted optimization for their infrastructure. With Pulumi Insights, companies can scan and sync their entire AWS resource inventory – including resources not managed with Pulumi IaC – ensuring a complete picture of their cloud assets. Compliance checks can be run against standards such as PCI DSS, ISO 27001, and CIS benchmarks, helping maintain regulatory compliance and mitigate risks. The AI-powered [Pulumi Copilot](/product/copilot/) provides an interactive assistant to gain visibility into a team's activity, discover cost saving opportunities, get compliant, and debug cloud failures. Pulumi Insights enables companies to discover, understand, manage, and improve their cloud infrastructure on their journey towards intelligent infrastructure management.
+
+### Pulumi ESC: Centralized Secrets Management
+
+Secrets sprawl, long-lived static credentials, and passing around secrets in plaintext to developers expose organizations to security vulnerabilities. [Pulumi ESC](/product/secrets-management/) addresses these challenges by offering seamless two-way integration – both pull and sync – with AWS Secrets Manager and other popular secrets stores, as well as the capability to consume secrets securely across any applications, tools, and CI/CD platform. Developers gain easy, secure access to secrets via CLI, API, Kubernetes operators, and SDKs – eliminating the need for .env files and reducing the security risks associated with them. Additionally, Pulumi Insights can issue dynamic, short-lived AWS authentication tokens on demand via OpenID Connect (OIDC), reducing the risks that come with long-lived access tokens, while RBAC, versioning, and a detailed audit log let you trust (and prove) your secrets are secure.
```

Comment:
Same here, gives a bit more context for why we launched it and connects it back to the founding vision

> This complexity creates significant challenges in managing sensitive credentials, API keys, and configurations across development, testing, and production environments. As organizations scale, they often face issues of secrets and configuration sprawl, duplication of secrets, and too many long-lived static secrets. None of the existing secrets management solutions adequately address these challenges, exposing enterprises to errors that can lead to security breaches, unintended exposure of sensitive data, and unauthorized access to critical resources.